src/components/containers/EventCard/EventCard.tsx

```diff
@@ -31,22 +38,72 @@ export const EventCard: React.FC<EventCardProps> = ({
         backgroundImage: `linear-gradient(90deg, ${color}a0, ${color}), url(${image})`,
     }
 
+    const afterEventCardStyle: React.CSSProperties = {
+        backgroundColor: `${color}`
+    }
+
+    console.log(color)
+
+
+    dayjs.locale('pl')
+    dayjs.extend(isBettwen)
```

Comment:
te rozszerzenia można ograć gdzieś przy głównym widoku, tak żeby nie robić tego przy każdym komponencie, proponuję wepchać to gdzieś do App.tsx